data/meson.build

```diff
@@ -58,6 +58,7 @@ endif
 
 blueprints = custom_target('blueprints',
   input: files(
+    'ui/add_style.blp',
```

Comment:
might not be scope of this pr, but perhaps sort this list alphabetically (Probably oversight on my end)